gnome-initial-setup/gis-driver.c

```diff
@@ -87,33 +87,55 @@ typedef struct _GisDriverPrivate GisDriverPrivate;
 
 G_DEFINE_TYPE_WITH_PRIVATE(GisDriver, gis_driver, GTK_TYPE_APPLICATION)
 
+/* Should be kept in sync with eos-installer */
 static gboolean
-running_live_session (void)
+check_for_live_boot (gchar **uuid)
```

Comment:
Oh I see... I was confused by the fact that it seemed optional from the first commit. In that case perhaps you want to assert that `uuid != NULL`?